diff -Nuard libglade-2.0.1.orig/glade/glade-gtk.c libglade-2.0.1/glade/glade-gtk.c
--- libglade-2.0.1.orig/glade/glade-gtk.c	2002-07-30 11:10:36.000000000 +0200
+++ libglade-2.0.1/glade/glade-gtk.c	2003-11-26 00:42:31.359886744 +0100
@@ -680,6 +680,7 @@
 	    !strcmp (childinfo->child->classname, "radio") ||
 	    !strcmp (childinfo->child->classname, "button")) {
 	    const char *label = NULL, *stock = NULL, *group_name = NULL;
+	    const char *tooltip = NULL;
 	    char *icon = NULL;
 	    gboolean use_stock = FALSE, active = FALSE, new_group = FALSE;
 	    gboolean use_underline = FALSE;
@@ -711,7 +712,7 @@
 		} else if (!strcmp (name, "visible")) {
 		    /* ignore for now */
 		} else if (!strcmp (name, "tooltip")) {
-		    /* ignore for now */
+		    tooltip = value;
 		} else if (!strcmp (name, "use_underline")) {
 		    use_underline = BOOL (value);
 		} else if (!strcmp (name, "inconsistent")) {
@@ -758,14 +759,14 @@
 		child = gtk_toolbar_append_element (
 		    GTK_TOOLBAR (parent),
 		    GTK_TOOLBAR_CHILD_TOGGLEBUTTON, NULL,
-		    label, NULL, NULL, iconw, NULL, NULL);
+		    label, tooltip, NULL, iconw, NULL, NULL);
 		gtk_toggle_button_set_active(
 		    GTK_TOGGLE_BUTTON (child), active);
 	    } else if (!strcmp (childinfo->child->classname, "radio")) {
 		child = gtk_toolbar_append_element (
 		    GTK_TOOLBAR (parent),
 		    GTK_TOOLBAR_CHILD_RADIOBUTTON, NULL,
-		    label, NULL, NULL, iconw, NULL, NULL);
+		    label, tooltip, NULL, iconw, NULL, NULL);
 
 		if (group_name) {
 		    g_object_set (G_OBJECT (child),
@@ -775,7 +776,7 @@
 	    } else
 		child = gtk_toolbar_append_item (
 		    GTK_TOOLBAR (parent),
-		    label, NULL, NULL, iconw, NULL, NULL);
+		    label, tooltip, NULL, iconw, NULL, NULL);
 	    
 	    /* GTK+ doesn't support use_underline directly, so we have to hack
 	       it. */
